src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -65,15 +66,15 @@ public JobHostConfiguration(string dashboardAndStorageConnectionString)
 
             AddService<IExtensionRegistry>(extensions);
             AddService<StorageClientFactory>(new StorageClientFactory());
-            AddService<INameResolver>(new DefaultNameResolver());
+            AddService<INameResolver>(new AppSettingOrEnvironmentNameResolver());
```

Comment:
Correct.

---
In reply to: [69617747](https://github.com/Azure/azure-webjobs-sdk/pull/760#discussion_r69617747) [](ancestors = 69617747)